src/core/include/cesium/omniverse/Context.h

```diff
@@ -1,166 +1,93 @@
 #pragma once
 
-#include "cesium/omniverse/RenderStatistics.h"
-#include "cesium/omniverse/SetDefaultTokenResult.h"
-#include "cesium/omniverse/TokenTroubleshooter.h"
-#include "cesium/omniverse/UsdNotificationHandler.h"
-
-#include <CesiumAsync/AsyncSystem.h>
-#include <CesiumGeospatial/Cartographic.h>
-#include <CesiumGeospatial/Ellipsoid.h>
-#include <CesiumGeospatial/GlobeAnchor.h>
-#include <CesiumGeospatial/LocalHorizontalCoordinateSystem.h>
-#include <glm/glm.hpp>
-#include <omni/fabric/SimStageWithHistory.h>
 #include <pxr/usd/usd/common.h>
-#include <spdlog/logger.h>
 
-#include <atomic>
 #include <filesystem>
 #include <memory>
 #include <vector>
 
+#include <gsl/span>
+
+namespace omni::fabric {
+class StageReaderWriter;
+}
```

Comment:
Super nit. For consistency, you might want a space after this line. I'm surprised clang-format doesn't add it.